source/administration/changelog.md

```diff
@@ -1,8 +1,8 @@
-# Mattermost Changelog
+# Mattermost Release Notes
 
-This changelog summarizes updates to [Mattermost Team Edition](https://mattermost.org/), an open source team messaging solution released monthly under an MIT license, and [Mattermost Enterprise Edition](https://mattermost.com/pricing-self-managed/), a commercial upgrade offering enterprise messaging for large organizations.
+These release notes summarize updates to [Mattermost Team Edition](https://mattermost.org/), an open source team messaging solution released monthly under an MIT license, and [Mattermost Enterprise Edition](https://mattermost.com/pricing-self-managed/), a commercial upgrade offering enterprise messaging for large organizations.
 
-Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
+Also see [changelogs in progress](https://bit.ly/2nK3cVf) for the next release.
```

Comment:
I left this as "changelogs" as I'm not certain what we should replace it with. Perhaps "PRs in progress" would work?

Reply:
I recommend spelling it out to read "pull requests in progress". This way it covers both technical and non-technical users; also aligns with the wording on the Github page.

Reply:
Do we need to open a separate ticket to make the changes to the LHS and breadcrumbs?
![image](https://user-images.githubusercontent.com/48559658/104232303-27e4f300-541e-11eb-93f3-ffcc3729af15.png)

Reply:
Those are good questions. I think they can all go into this PR as they're related.